Clarify expected return value of first assignment (in Recursion Project lesson) (#22329)
TheOdinProject/curriculum: ruby_programming/computer_science/project_recursion.md

### Assignment 1

<div class="lesson-content__panel" markdown="1">

>   1. Using iteration, write a method `#fibs` which takes a number and returns that many members of the fibonacci sequence. It should return an array.

Reviewer: ```suggestion
  1. Using iteration, write a method `#fibs` which takes a number and returns an array containing that many numbers from the fibonacci sequence.
```

Author: That makes perfect sense, thank you. I made the change.
